addons/payment/views/payment_token_views.xml

```diff
@@ -33,7 +33,7 @@
         <field name="name">payment.token.list</field>
         <field name="model">payment.token</field>
         <field name="arch" type="xml">
-            <tree string="Payment Tokens">
+            <tree string="Payment Tokens" create="false">
                 <field name="payment_details"/>
                 <field name="partner_id"/>
                 <field name="provider_id" readonly="1"/>
```

Comment:
You could remove the `readonly="1"` on both fields here, it's confusing imho (since it's not allowed for any field anyway)

Reply:
Yep, true